config/config.go

```diff
@@ -36,15 +33,12 @@ func Get() (*Config, error) {
 
 	cfg = &Config{
 		BindAddr:                  ":23100",
-		Brokers:                   []string{"localhost:9092"},
 		DatasetAPIURL:             "http://localhost:22000",
 		DatasetAPISecretKey:       "FD0108EA-825D-411C-9B1D-41EF7727F465",
 		ElasticSearchAPIURL:       "http://localhost:9200",
 		GracefulShutdownTimeout:   5 * time.Second,
 		HealthCheckInterval:       1 * time.Minute,
 		HealthCheckTimeout:        2 * time.Second,
-		HierarchyBuiltTopic:       "hierarchy-built",
-		KafkaMaxBytes:             2000000,
```

Comment:
these needs to be removed from README.md
also, search for `searchIndexProducer` to remove that

Reply:
Always forget the readme 😄 
Good spots - I have removed them. I also snuck in a default config change for elastic search request signing 👍